### PR TITLE
net-p2p/retroshare: avoid broken doxygen-1.8.16

### PR DESCRIPTION
**Pull request by G10h4ck**
Fix bug https://bugs.gentoo.org/show_bug.cgi?id=709252 and
  https://bugs.gentoo.org/699164 due to doxygen 1.8.16 bug
  https://github.com/doxygen/doxygen/issues/7236

### CODE REVIEW

**Comment by a17r** [comment]
A bug in doxygen should be fixed in doxygen, not in x consumer packages.

Did you file a bug against `app-doc/doxygen` in Gentoo Bugzilla?

**Comment by G10h4ck** [comment]
Done https://bugs.gentoo.org/709316 but I doubt doxygen-1.8.16 ebuild will be removed soon from gentoo, so better retroshare avoid that, also that bug doesn't seems to affect so critically other packages

**Comment by G10h4ck** [comment]
Also it is an incompatibility with a specific doxygen version, so please merge this PR that just avoid RS being compiled with an incompatible doxygen version

**Comment by juippis** [comment]
Also the first bug you reference in your commit message seems irrelevant to this PR, maybe you're looking for some other bug?

**Comment by G10h4ck** [comment]
@juippis the bug you are experiencing now is due to another issue that is addressed in it's own PR https://github.com/gentoo/gentoo/pull/14660 to have retroshare compiling again both PRs need to be merged, as I am RetroShare proxy maintainer I wonder if there is a way to get patches into gentoo faster... those too are quite trivial fixes and it is already more then one months int the queue...

File: net-p2p/retroshare/retroshare-0.6.5.ebuild [hunk 1]
```diff
@@ -50,7 +50,7 @@ RDEPEND="
 DEPEND="${RDEPEND}
 	gui? ( dev-qt/designer:5 )
 	jsonapi? (
-		app-doc/doxygen
+		|| ( <app-doc/doxygen-1.8.16 >=app-doc/doxygen-1.8.17 )
```
Comment (a17r) [inline net-p2p/retroshare/retroshare-0.6.5.ebuild]: please note that newer version should always come first here.

Comment (G10h4ck) [reply net-p2p/retroshare/retroshare-0.6.5.ebuild]: thanks, I'll take it in acocunt for next PR ;)

**Pull request closed without merging**